src/test/pybind/test_rados.py

```diff
@@ -598,6 +598,25 @@ def test_locator(self):
         eq(objects, [])
         self.ioctx.set_locator_key("")
 
+    def test_operate_aio_write_op(self):
+        lock = threading.Condition()
+        count = [0]
+        def cb(blah):
+            with lock:
+                count[0] += 1
+                lock.notify()
+            return 0
+        write_op = self.ioctx.create_write_op()
```

Comment:
could you use `WriteOpCtx` instead?

Reply:
@tchaikov   Yes, i have tested it.

Reply:
@ tchaikov  I already deferred to your suggestion to revise the code, please re-confirm.